### PR TITLE
feat: `ProfileWindow` now shows Visits as Cells instead of raw text

### PR DESCRIPTION
**Pull request by Q-gabe**
Enhancement to `ProfileWindow` as suggested in #121.

**Changes**:
- `ProfileVisitCard` now represents a single Visit instance in a ListView in `ProfileWindow`.
- Refactored `ProfileWindow` to delegate String processing to a separate `ProfileUtil.java` in `/commons/util/`.

### CODE REVIEW

File: src/main/java/seedu/address/ui/ProfileVisitCard.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        this.report = report;
+        this.date = report.date;
+        this.diagnosis = report.getDiagnosis();
+        this.medication = report.getMedication();
+        this.remarks = report.getRemarks();
+
+        // Set date
+        profileVisitDate.setText("Visitation Report on  [" + date + "]");
+
+        // Set Diagnosis
```
Comment (Wingedevil) [inline src/main/java/seedu/address/ui/ProfileVisitCard.java]: Maybe we can SLAP this a bit more?

Comment (Q-gabe) [reply src/main/java/seedu/address/ui/ProfileVisitCard.java]: Are you referring to line 45-50?

Comment (Q-gabe) [reply src/main/java/seedu/address/ui/ProfileVisitCard.java]: I can perhaps refactor this whole setting of labels outside of the constructor, for the next iteration of refactoring.

Comment (Wingedevil) [reply src/main/java/seedu/address/ui/ProfileVisitCard.java]: Actually more like setting diagnosis, medication and remarks, extract the method out

Comment (Q-gabe) [reply src/main/java/seedu/address/ui/ProfileVisitCard.java]: Yeah thats actually a good idea. I'll do it in the next refactor run.

**Pull request merged**